.gitignore

```diff
@@ -77,9 +77,7 @@ Cargo.lock
 
 # === PROJECT SPECIFIC ===
 # mc decompiled source
-mc-source-code/DecompilerMC/
-mc-source-code/src/
-mc-source-code/target/
+mc-source-code/
```

Comment:
Why you modified the .gitignore ?

Reply:
Sorry I've used the "decompile.sh" file and that folder for save researches, to prevent any accidental upload I've changed to include any file and subfolder